res/values/strings.xml

```diff
@@ -422,9 +422,9 @@
     <string name="pref_language">Language</string>
     <string name="pref_incognito_keyboard">Incognito keyboard</string>
     <!-- Translators: Keep in mind that this is a Request - it must be clear in the wording that this cannot be enforced. -->
-    <string name="pref_incognito_keyboard_explain">Request keyboard to disable personalized learning</string>
+    <string name="pref_incognito_keyboard_explain">Request keyboard to turn off personalized learning</string>
     <string name="pref_read_receipts">Read receipts</string>
```

Comment:
This is problematic.

Reply:
in general, i am not sure if "turn off" is better than "disable". but my english is not good enough to decide that.

Reply:
What did you learn first? Disable/enable is only additionally meaningful if it can be enabled and not on.